sda-admin/dataset/dataset.go

```diff
@@ -0,0 +1,59 @@
+package dataset
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"path"
+
+	"github.com/neicnordic/sensitive-data-archive/sda-admin/helpers"
+)
+
+type RequestBodyDataset struct {
+	AccessionIDs []string `json:"accession_ids"`
+	DatasetID    string   `json:"dataset_id"`
+}
+
+// Create creates a dataset from a list of accession IDs and a dataset ID.
+func Create(apiURI, token, datasetID string, accessionIDs []string) error {
+	parsedURL, err := url.Parse(apiURI)
+	if err != nil {
+		return err
+	}
+	parsedURL.Path = path.Join(parsedURL.Path, "dataset/create")
+
+	requestBody := RequestBodyDataset{
+		AccessionIDs: accessionIDs,
+		DatasetID:    datasetID,
+	}
+
+	jsonBody, err := json.Marshal(requestBody)
+	if err != nil {
+		return fmt.Errorf("failed to marshal JSON, reason: %v", err)
+	}
+
+	_, err = helpers.PostRequest(parsedURL.String(), token, jsonBody)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Release releases a dataset for downloading
+func Release(apiURI, token, datasetID string) error {
+	parsedURL, err := url.Parse(apiURI)
+	if err != nil {
+		return err
+	}
+	parsedURL.Path = path.Join(parsedURL.Path, "dataset/release") + "/" + datasetID
+
+	jsonBody := []byte("{}")
+	_, err = helpers.PostRequest(parsedURL.String(), token, jsonBody)
```

Comment:
```suggestion
	_, err = helpers.PostRequest(parsedURL.String(), token, nil)
```

Reply:
Since PostRequest is defined as 
```golang
func PostReq(url, token string, jsonBody []byte) ([]byte, error) {
```
isn't it better to specify the third parameter as `[]byte("{}")` than `nil`?

Reply:
That's because how you defined the function.
An empty byte slice is the same as `nil` and since the dataset release doesn't require any payload you can just as well omit it by sending`nil` here.

Reply:
OK. fixed in the commit 95c6bf9